### PR TITLE
add va.gov robot.txt

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
- the existing va.gov robots.txt has a listing of the VBA decisions, which are not actually part of the site and now have an alternative UI
- vets.gov (via Metalsmith) already has a robots.txt and will need to be updated with what's necessary for the merged site

## Testing done
verified locally 

## Screenshots vets.gov
screen shot 2018-10-09 at 15 07 26

## Screenshots va.gov
screen shot 2018-10-10 at 13 46 25

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: script/options.js [hunk 1]
```diff
@@ -66,9 +66,14 @@ function applyEnvironmentOverrides(options) {
   switch (options.buildtype) {
     case environments.DEVELOPMENT:
     case environments.STAGING:
+      options.move = [{ source: 'vets-robots.txt', target: 'robots.txt' }];
+      options.remove = ['va-robots.txt'];
       break;
 
     case environments.PRODUCTION:
+      options.move = [{ source: 'vets-robots.txt', target: 'robots.txt' }];
+      options.remove = ['va-robots.txt'];
```
Comment (rianfowler) [inline script/options.js]: figured i wouldn't dry this out for now since we're just going to remove it when vets.gov is deprecated.

**Comment by rianfowler** [comment]
Not at expert at robots.txt- assuming for va.gov that we want to merge what's existing with what's on vets.gov

**Comment by wyattwalter** [comment]
Need to be careful with the merge on this one. I believe that once you deploy this search.gov will start indexing that https://www.va.gov/sitemap.xml which as you pointed out has VBA decisions that I don't think we want indexed. Ideally, it could be pointed at preview.va.gov/sitemap.xml until launch, but I don't know how much time we'd need for indexing before launch.

**Comment by rianfowler** [comment]
I just copied the instructions from the ticket- what is a VBA decision @wyattwalter ?

**Review by cehsu — APPROVED** [review]
LGTM.

**Comment by BBarone1** [comment]
Hey echoing what Wyatt said above. We should be pointing this to preview.va.gov for indexing, NOT www.va.gov. Obviously www.va.gov is currently a different site in production that we don’t own, and we don’t want to be serving up that index to our search functionality. 

We’ll have to switch it over to www.va.gov post launch. Let me know if any questions with that

File: assets/va-robots.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+User-agent: Synapse
+Disallow:/
+
+User-Agent: *
+Disallow: /analytics-opt-out.html
+Disallow: /cgi-bin/
+Disallow: /provideinfo/includes/
```
[the remark this answers is not in the document]
Comment (rianfowler) [reply assets/va-robots.txt]: these are pages that exist in the `va.gov' robots.txt. I'm removing them for now. I'll note in the followup ticket that these should be merged when we take over va.gov

File: assets/va-robots.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+User-agent: Bingbot
+Disallow: /
+
+User-agent: Slurp
+Disallow: /
+
+User-agent: DuckDuckBot
+Disallow: /
+
+User-Agent: *
```
Comment (rianfowler) [inline assets/va-robots.txt]: I'm not search what search.gov needs to index this page- i assume disallowing * will prevent it from indexing preview.va.gov. 

Opted to specifically target the top 4 search engines- should make this easier to update when we launch as well.

**Comment by rianfowler** [comment]
@wyattwalter I removed these from the preview.va.gov robots.txt: 

```
Disallow: /cgi-bin/
Disallow: /provideinfo/includes/
```

I'd pulled them from the current va.gov robots.txt and I assume they're not relevant. Do you know if we need to have these disallows when we launch va.gov?

**Comment by rianfowler** [comment]
updated VA.gov screenshot

**Comment by wyattwalter** [comment]
We are going to be proxying back to the existing va.gov, so we still want to block that stuff if those paths still exist.

**Comment by rianfowler** [comment]
Thanks @wyattwalter. I restored one of the disallows since I was able to hit it in the browser. `/provideinfo/includes/` 404s in browser / curl.

**Pull request merged**